test/integration/analytical_policy.hpp

```diff
@@ -1547,7 +1547,7 @@ void test_analytical_oregonator(
 
   auto state = solver.GetState();
 
-  state.SetRelativeTolerance(1e-8);
+  state.SetRelativeTolerance(1e-6);
```

Comment:
@K20shores why we need to change the relative tolerance here?

Reply:
Oregonator for the cuda solver was running 30 seconds slower than on main. When I looked into it, it seems that [main uses `1e-6`](https://github.com/NCAR/micm/blob/main/test/integration/cuda/test_cuda_analytical_rosenbrock.cpp#L246). I guess when we translated to the new API we accidentally set the tolerance incorrectly. So this change really just uses what's already on main.

Reply:
Oh perfect. Thanks for the clarification.